src/library/Hooks/useTimeLeft/index.tsx

```diff
@@ -14,10 +14,7 @@ import {
   TimeLeftRaw,
 } from './types';
 
-export const useTimeLeft = ({
-  refreshCallback,
-  refreshInterval,
-}: TimeleftHookProps) => {
+export const useTimeLeft = (TimeleftHook?: TimeleftHookProps) => {
```

Comment:
`TmeleftHook` is not a hook, they can be treated as props to the hook. Therefore this should just be called `props`.